phone/src/apps/twitter/components/ModalBackground.tsx

```diff
@@ -0,0 +1,25 @@
+import { memo } from 'react';
+import makeStyles from '@mui/styles/makeStyles';
+import { useModal } from '../hooks/useModal';
+
+const useStyles = makeStyles(() => ({
+  backgroundModal: {
+    background: 'black',
+    opacity: '0.6',
+    position: 'absolute',
+    top: 0,
+    left: 0,
+    right: 0,
+    bottom: 0,
+    zIndex: 5,
+  },
+}));
+
+const ModalBackground = () => {
```

Comment:
Fine for now, but we should used a styled div for this. Introduced in the message PR :)